src/main/java/io/jenkins/plugins/opentelemetry/semconv/JenkinsSemanticMetrics.java

```diff
@@ -7,6 +7,7 @@
 
 public class JenkinsSemanticMetrics {
     public static final String  CI_PIPELINE_RUN_ACTIVE =            "ci.pipeline.run.active";
+    public static final String  CI_PIPELINE_RUN_DURATION =          "ci.pipeline.run.duration";
```

Comment:
This matches the other metrics currently defined in this plugin (use of `ci.*` namespace).
I've checked against https://opentelemetry.io/docs/specs/semconv/attributes-registry/cicd/ and https://github.com/open-telemetry/semantic-conventions/blob/e63b2cfd377688c3f0a9054636fb52d973998a7a/docs/cicd/cicd-metrics.md from PR https://github.com/open-telemetry/semantic-conventions/pull/1383 to verify if that name is likely to be accepted in the OpenTelemetry semantic conventions.
Currently it looks more likely that the `cicd.*` namespace will be used in the semantic conventions.
However that is a discussion for another day. This PR #959 looks good as it is.

Reply:
Thanks. I'm aligned with you, I also propose to move forward with the `ci.*` namespace and rename once OTel has standardized the namespace and hopefully the metrics we implemented 🤞